pyproject.toml

```diff
@@ -1,15 +1,15 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm"]
+requires = ["setuptools", "setuptools-scm", "requests", "numpy", "pandas", "lxml", "matplotlib"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "ifk-scb-compilations"
+name = "ifk_scb_compilations"
 version = "0.1.0"
 description = "Unspecified."
```

Comment:
We could add some description here (and the same in the copier file).

Reply:
added.